client/src/components/Footnote.js

```diff
@@ -19,7 +19,12 @@ const StyledFooter = styled.footer`
   line-height: 2rem;
   font-family: Poppins;
   font-size: 1.1rem;
-  margin-top: 3rem;
+
+  @media screen and (min-width: ${mq.tablet.wide.minWidth}) {
```

Comment:
I think it's at this media query but for some reason the footer isn't all the way at the bottom of the page for some pages when width is less than 1024px

<img width="1063" alt="Screen Shot 2020-05-10 at 11 04 44 PM" src="https://user-images.githubusercontent.com/42228386/81529216-3caf3b00-9313-11ea-8654-33a291cddc6d.png">
<img width="1071" alt="Screen Shot 2020-05-10 at 11 05 18 PM" src="https://user-images.githubusercontent.com/42228386/81529228-40db5880-9313-11ea-9157-3b2a27c9bc92.png">

Reply:
I think we need some spacing between the footer and everything else, though in some pages it looks fine

<img width="381" alt="Screen Shot 2020-05-10 at 11 10 40 PM" src="https://user-images.githubusercontent.com/42228386/81529365-931c7980-9313-11ea-8a6c-dd469a9de36c.png">

Reply:
There is something wrong between width: 450px and 1024px, e.g. the content (the "A place to give and get help...") is missing. I don't think this is a footer issue, I think our main containers for various pages have different min-widths... I tried to fix it by setting a min-width to the Main component in /templates/layouts/main since that is rendering the different pages but it doesn't seem to fix the issue between 450px and 1024px...